.env.local

```diff
@@ -0,0 +1,48 @@
+ALLOWED_HOSTS=localhost
```

Comment:
est-ce que `.env.local` est le bon nom ? On veut pas prendre precedence des `.env`

Reply:
un packet qui pourrait être utile pour faire vivre des configs Docker et "local" : https://pypi.org/project/django-environ/

Reply:
on utilise [dotenv](https://github.com/jpadilla/django-dotenv) et je viens de voir que ça utilise `.env` par défaut, alors normalement pas de pb pour vous.
Pour [Vue 2](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables) et [Vue 3](https://vite.dev/guide/env-and-mode#env-files) c'est possible que `.env.local` va prendre precedence, alors je vais renommer le fichier `.env.docker` pour être certaine